wpimath/src/main/java/edu/wpi/first/math/system/plant/DCMotor.java

```diff
@@ -250,4 +250,40 @@ public static DCMotor getRomiBuiltIn(int numMotors) {
     return new DCMotor(
         4.5, 0.1765, 1.25, 0.13, Units.rotationsPerMinuteToRadiansPerSecond(150.0), numMotors);
   }
+
+  /**
+   * Return a gearbox of Kraken X60 brushless motors.
+   *
+   * @param numMotors Number of motors in the gearbox.
+   * @return a gearbox of Kraken X60 motors.
+   */
+  public static DCMotor getKrakenX60(int numMotors) {
+    // From https://store.ctr-electronics.com/announcing-kraken-x60/
+    return new DCMotor(
+        12, 7.09, 366, 2, Units.rotationsPerMinuteToRadiansPerSecond(6000), numMotors);
+  }
+
+  /**
+   * Return a gearbox of Kraken X60 brushless motors with FOC (Field-Oriented Control) enabled.
+   *
+   * @param numMotors Number of motors in the gearbox.
+   * @return A gearbox of Kraken X60 FOC enabled motors.
+   */
+  public static DCMotor getKrakenX60FOC(int numMotors) {
```

Comment:
The Falon500 FOC method only has the F capitalized, this should also match that or that should be changed to match this.

Reply:
C++ uses uppercase while Java uses lowercase (#juststyleguidethings). The Java one should use "Foc", but the C++ one is fine.